.changelog/unreleased/features/242-change-return-value.md

```diff
@@ -0,0 +1,2 @@
+- Change host_height, host_timestamp return value in ics02, ics03, ics04
```

Comment:
Additionally, can we move this to the `.changelog/unreleased/breaking-changes` directory?